ql/cashflows/overnightindexedcoupon.cpp

```diff
@@ -67,7 +71,7 @@ namespace QuantLib {
                 }
 
                 // today is a border case
-                if (i<n && fixingDates[i] == today) {
+                if (i < n && fixingDates[i] == minDate) {
```

Comment:
This check need to stay on `today` (it discriminates between past and future fixings).

Reply:
Eh? What if the passed date is before `today`?